imagemin.js

```diff
@@ -5,7 +5,7 @@ const GIFImages = 'src/images/**/*.gif';
 const output = 'images';
 
 const imageminJpegtran = require('imagemin-jpegtran');
-const imageminJpegoptim = require('imagemin-jpegoptim');
+// const imageminJpegoptim = require('imagemin-jpegoptim');
```

Comment:
Ideally, any commented out code should be removed.

Reply:
uncommented the line - was added for debugging and not reversed